### ISSUE TITLE
Suggest to create "symmetric view" in the linked table when adding a many to many relationship

### ISSUE THREAD

**Issue by asitemade4u** [directus/directus#2636]
# Feature Request
Before I chose Directus, I had a thorough look at Strapi. They have developed an interesting feature that would also be helpful in Directus: when adding a many to many relationship between, say, table A and B:

- the related records of B are shown in records of table A (actual behavior in Directus)
- but the records of table B also show linked records from table A (not available in Directus)

At least, this should be an option in Directus.

 
## What problem does this feature solve?
In a many to many relationship between tables, there is no _hierarchy_ between them: they are equally weighted.Therefore there is no reason to favor one against the other by displaying the related records in one, but not in the other.

## How do you think this should be implemented?
- either by automatically creating the symmetric structure in the related table
- or at least suggest to create it

**Comment by rijkvanzanten** [comment]
> but the records of table B also show linked records from table A (not available in Directus)

You can add a many to many interface to table B with the same relationship to achieve this

**Comment by asitemade4u** [comment]
Yes I know, but having it suggested by Directus would be great -- just 
as the great feature you implemented in the many to many relationship 
creation process: the checkbox that says *Auto-Generate*.

**Comment by benhaynes** [comment]
I agree — I think it would be nice to include an auto-generate the "corresponding" field on the reverse collection. I'll add this to the v9 design now.
